system/Debug/Toolbar/Views/toolbar.js

```diff
@@ -70,7 +70,24 @@ var ciDebugBar = {
         // Hook up generic toggle via data attributes `data-toggle="foo"`
         var links = this.toolbar.querySelectorAll("[data-toggle]");
         for (var i = 0; i < links.length; i++) {
-            links[i].addEventListener("click", ciDebugBar.toggleRows, true);
+            let toggleData = links[i].getAttribute("data-toggle");
+            if(toggleData=="datatable"){
```

Comment:
```suggestion
            if (toggleData === "datatable") {
```
Please fix styling of the code - for the entire code block (mostly spaces)